tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -191,6 +186,16 @@ func (s *stresser) Stress() error {
 	}
 	s.stressTable = createStressTable(stressEntries)
 
+	return nil
+}
+
+func (s *stresser) Stress() error {
+	if err := s.setup(); err != nil {
```

Comment:
I move the code in the setup back to stress.
Also for the liveness stuff. I can have the map `expiring map[LeaseID]time.Time` where i just create the lease once and put them in there with no keepalive at all. I would expected it to be expired after the deadline. In this way, i don't have the  deal with the leases in the alive map. However, I am unsure if this is a good test case since I don't pick lease out of the live lease map whereas picking lease out of live lease map adds more variation due to the live lease could have survived more faults. In addition, I am thinking about having another map which is keepalive map with no lease drop, so i would check every time to make sure that there aren't any lease drop in that map. what do you thinking? @heyitsanthony